index.js

```diff
@@ -134,6 +143,47 @@ exports.execute = async (srcName, query, params = {}, options = {}) => {
   }
 };
 
+/**
+ * Execute multiple inserts in batch
+ * @param {*} srcName 
+ * @param {*} query 
+ * @param {*} binds 
+ * @param {*} options 
+ */
```

Comment:
Could you include in the description the documentation reference for the oracledb `executeMany` method and/or provide also the parameter types associated to each parameter? Especially with binds (whether we need to provide one object, or array of objects or array of arrays of the values parameterized on each query). 

Also, I assume it is queries, instead of queries. 

Other than that, everything looks good to me.

Reply:
added jsdoc with examples